compiler/qsc_lowerer/src/lib.rs

```diff
@@ -728,6 +728,18 @@ impl Lowerer {
             _ => self.exec_graph.push(ExecGraphNode::Expr(id)),
         }
 
+        if matches!(
```

Comment:
Why not make this park of the match above?

Reply:
I find the single match a bit clearer. That way I see that Expression Id is actually pushed instead of relying on fairly obscure "_" case before that.

Reply:
Seems reasonable. I'll make the change.